data/fedora-update.yaml

```diff
@@ -359,6 +359,11 @@ python-pydns:
   status: dropped
   note: |
     Recommended replacement: `python3-py3dns`
+python2-pytest:
+  status: dropped
+  note: |
+    This is the Python 2 version.
+    The `python-pytest` package provides `python3-pytest`.
```

Comment:
```suggestion
    The `pytest` package provides `python3-pytest`.
```

Reply:
Whoops! Nice catch

Reply:
I wouldn't have noticed this if it was not me who made the split.